Kickstarter-iOS/Features/MessageBanner/Views/MessageBannerViewTests.swift

```diff
@@ -0,0 +1,52 @@
+@testable import Kickstarter_Framework
+// @testable import KsApi
```

Comment:
looks like we can remove this import for now

Reply:
Good catch, thanks!